src/bundle/Resources/views/Limitation/udw_limitation_value_tag.html.twig

```diff
@@ -0,0 +1,17 @@
+<li class="mt-2">
```

Comment:
This part: `<li class="mt-2">` has to be moved to the parent template. The component itself is located in `<div class="ez-tag" ...>`

Reply:
I renamed `src/bundle/Resources/views/Limitation/udw_limitation_value_tag.html.twig` to `udw_limitation_value_list_item.html.twig`. I have put `<li class="mt-2">` here because we are passing this twig in data-attribute as template and we want _li_ element in this template.